src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -828,7 +828,7 @@ public static void EndsWith(string s, string value, StringComparison comparisonT
         [InlineData("abcdefghijklmnopqrstuvwxyz", 'z', true)]
         public static void EndsWith(string s, char value, bool expected)
         {
-            Assert.Equal(expected, s.EndsWith(value));
+            Assert.Equal(expected, s.EndsWith(value.ToString()));
```

Comment:
The whole point of this test is to test the new `EndsWith(char)` overload. Same for the `StartsWith(char)` test below. #16198